testing/integration/testdata/.upgrade-test-agent-versions.yml

```diff
@@ -8,5 +8,5 @@ testVersions:
   - 8.16.4
   - 8.15.5
   - 8.14.3
+  - 7.17.28
   - 7.17.28-SNAPSHOT
```

Comment:
I would have expected to see 7.17.29-SNAPSHOT instead of 7.17.28-SNAPSHOT here :thinking:

Reply:
I would have expected the same.